Library/Formula/kobalt.rb

```diff
@@ -0,0 +1,48 @@
+
+class Kobalt < Formula
+  desc "A build system"
+  homepage "http://beust.com/kobalt"
+  url "https://github.com/cbeust/kobalt/releases/download/0.633/kobalt-0.633.zip"
+  version "0.633"
+  sha256 "bb468a7b8761de20c4700e18a6de55ee0712edd0e9d04748e53592c91389c94e"
+
+  def install
+    prefix.install "kobaltw"
+    prefix.install "kobalt"
+
+    # kobaltw expects the jar file to be in ./kobalt/wrapper so
+    # install it to /usr/local/bin manually and correct the
+    # jar location
+    kobaltw = "/usr/local/bin/kobaltw"
```

Comment:
Yes. You can’t assume anything on the Homebrew location; people can install it outside of `/usr/local`.

Regarding the `./kobalt/wrapper` issue, you might want to try something like this instead:
```rb
  def install
    libexec.install %w[kobaltw kobalt]
    kobaltw = libexec/"kobaltw"
    kobaltw.chmod 0755
    bin.write_exec_script kobaltw
  end
```